src/main/java/com/datasonnet/spi/DataFormatService.java

```diff
@@ -1,16 +1,14 @@
-package com.datasonnet.portx.spi;
+package com.datasonnet.spi;
 
 import java.util.*;
 
 public class DataFormatService {
 
     private static DataFormatService service;
 
-    private static Map<String, List<DataFormatPlugin>> pluginRegistry;
+    private Map<String, List<DataFormatPlugin>> pluginRegistry = new HashMap<>();
```

Comment:
We already have a shared instance, having a shared pluginRegistry just makes it harder to test in isolation